### PR TITLE
Do not add validation errors for FacilityAccount#new

### PR DESCRIPTION
**Pull request by jhanggi**
https://www.pivotaltracker.com/story/show/115368465

This is an issue in NU

### CODE REVIEW

**Comment by drench** [comment]
:+1:

File: app/services/nufs_account_builder.rb [hunk 1]
```diff
@@ -31,4 +24,18 @@ def after_build
...
+  def load_account_components
+    # if we're loading the builder in a #new action, this won't be populated yet
+    return unless account.account_number
+
+    # This will populate virtual fields like fund, dept
+    account.load_components
+    rescue AccountNumberFormatError
+      # do nothing
+    rescue ValidatorError => e
+      account.errors.add(:base, e.message)
```
Comment (benlinton) [inline app/services/nufs_account_builder.rb]: Indenting for `rescue` without `begin`?

File: spec/services/nufs_account_builder_spec.rb [hunk 2]
```diff
@@ -20,30 +20,42 @@
...
-          account_number: "1234",
-          description: "foobar",
-        }
-      )
-    end
-
-    it "sets expired_at" do
-      expect(builder.build.expires_at).to be_present
-    end
+    describe "without params (like FacilityAccounts#new" do
```
Comment (benlinton) [inline spec/services/nufs_account_builder_spec.rb]: Missing `)`?

**Comment by benlinton** [comment]
:+1:

File: app/services/nufs_account_builder.rb (hunk 1 above)
Comment (jhanggi) [inline app/services/nufs_account_builder.rb]: Good catch.

**Pull request merged**